app/scripts/modules/kubernetes/serverGroup/configure/wizard/basicSettings.html

```diff
@@ -40,6 +40,17 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">
+      TERM Grace Period
+      <help-field key="kubernetes.serverGroup.terminationGracePeriodSeconds"></help-field>
+    </div>
+    <div class="col-md-2"><input type="number" min="0"
```

Comment:
Actually, this doesn't feel like a good spot for this field. Could it live under "Advanced Settings"?